provider/providers/ipvsdr/ipvsdr.go

```diff
@@ -522,28 +532,23 @@ func (f *ipvsdr) generateDeployment(lb *netv1alpha1.LoadBalancer) *extensions.De
 									Value: lb.Name,
 								},
 							},
-							// TODO
-							// Args: []string{
-							// // watch on which lb
-							// },
-							// ReadinessProbe: &v1.Probe{
-							// 	Handler: v1.Handler{
-							// 		HTTPGet: &v1.HTTPGetAction{
-							// 			Path:   "/ingress-controller-healthz",
-							// 			Port:   intstr.FromInt(10254),
-							// 			Scheme: v1.URISchemeHTTP,
-							// 		},
-							// 	},
-							// },
-							// LivenessProbe: &v1.Probe{
-							// 	Handler: v1.Handler{
-							// 		HTTPGet: &v1.HTTPGetAction{
-							// 			Path:   "/ingress-controller-healthz",
-							// 			Port:   intstr.FromInt(10254),
-							// 			Scheme: v1.URISchemeHTTP,
-							// 		},
-							// 	},
-							// },
+							VolumeMounts: []v1.VolumeMount{
+								{
+									Name:      "modules",
+									MountPath: "/lib/modules",
+									ReadOnly:  true,
+								},
+							},
+						},
+					},
+					Volumes: []v1.Volume{
+						{
+							Name: "modules",
+							VolumeSource: v1.VolumeSource{
+								HostPath: &v1.HostPathVolumeSource{
+									Path: "/lib/modules",
```

Comment:
/lib/modules for 32-bit
/lib64/modules for 64-bit
Do you need /lib/ certainly?